lib/matplotlib/font_manager.py

```diff
@@ -929,6 +940,45 @@ def set_fontconfig_pattern(self, pattern):
             else:
                 getattr(self, "set_" + key)(val)
 
+    def get_math_fontfamily(self):
+        """
+        Return the name of the font family used for math text.
+        The default font is :rc:`mathtext.fontset`.
```

Comment:
Minor formatting: Docstrings should have one summary sentence, which is separated from all following text by a blank line (c.f. https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)
```suggestion
        Return the name of the font family used for math text.

        The default font is :rc:`mathtext.fontset`.
```